packages/server-wallet/src/protocols/__test__/defund-channel.test.ts

```diff
@@ -204,21 +204,6 @@ describe('when the channel is finalized on chain', () => {
   });
 });
 
-it('should fail the objective when the channel does not exist', async () => {
```

Comment:
This was removed due to the fix in 1916f96. Now that a `channelId` can be extracted by objective model, it will stop an objective from being inserted with channelId not in the channel table, so we no longer need to worry about a channel not existing.